build_scripts/build_usd.py

```diff
@@ -97,6 +97,19 @@ def Python3():
     return sys.version_info.major == 3
 
 def GetLocale():
+    if Windows():
+        # Windows handles encoding a little differently then Linux / Mac
+        # For interactive streams (isatty() == True) it will use the
+        # console codepage, otherwise it will return an ANSI codepage.
+        # To keep things consistent we'll force UTF-8 for non-interactive
+        # streams (which is recommended by Microsoft). See:
+        # https://docs.python.org/3.6/library/sys.html#sys.stdout
+        # https://docs.microsoft.com/en-us/windows/win32/intl/code-page-identifiers
```

Comment:
I was able to track down the build issue with Windows Azure and it has something to do with what default encoding is set for the terminal running `build_usd.py`. I made a small change to [GetLocale() in build_usd.py](https://github.com/PixarAnimationStudios/USD/pull/1987/files#diff-88086ae4493b825e88542e8e0f512ad651197ae226692311f4e15699c9275ad8R101) to force `UTF-8`  on non-interactive terminals for Windows.

Specifically [sys.stdout.encoding](https://docs.python.org/3.6/library/sys.html#sys.stdout) on Windows will use the ANSI codepage for non-interactive streams and [Microsoft recommends to use UTF-8 or UTF-16 over ANSI codepages](https://docs.microsoft.com/en-us/windows/win32/intl/code-page-identifiers)

Reply:
nice find